src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig

```diff
@@ -90,12 +90,44 @@
                 <td><code>{{ message.domain }}</code></td>
                 <td>
                     <code>{{ message.id }}</code>
+
                     {% if message.count > 1 %}<br><small style="color: gray;">(used {{ message.count }} times)</small>{% endif %}
+                    {% if message.transChoiceNumber is not null %}<br><small style="color: gray;">(use pluralization)</small>{% endif %}
+
+                    <div>
+                        [<a href="#" onclick="return openParameters(this);" style="text-decoration: none;"
```

Comment:
`message.parameters` is never empty, and reflect all the calls to the given translation key. So it's perfectly possible to have:

- one first call with params: `{"%foo%": "bar"}`
- one second call without: `null`
- one third with other params: `{"%bar%": "foo"}`

If it's called 5 times without parameters, it will display 5 times `null`.